sympy/core/power.py

```diff
@@ -1212,6 +1212,22 @@ def _is_one(expr):
                 or self.base.is_irrational):
                 return self.exp.is_rational
 
+    def _eval_is_transcendental(self):
+        # Gelfond-Schneider theorem
+        def _is_one(expr):
+            try:
+                return (expr - 1).is_zero
+            except ValueError:
+                # when the operation is not allowed
+                return False
+
+        if self.base.is_zero or _is_one(self.base):
+            return False
+        elif self.base.is_real and self.exp.is_irrational:
```

Comment:
According to the theorem, this should be `is_algebraic`, not `is_real`.

Reply:
Yes, the issue is the theorem happens to use a different meaning for algebraic (including complex values). [Here](https://en.wikipedia.org/wiki/Gelfond%E2%80%93Schneider_theorem).